src/modules/presence/doc/presence_admin.xml

```diff
@@ -1284,6 +1284,27 @@ pres_update_watchers("sip:test@kamailio.org", "presence");
 ...
 </programlisting>
     </section>
+	
+<section>
+	<section id="presence.r.htable_db_restore">
+	  <title>presence.htable_db_restore</title>
+	  <para>
+		Manually apply changes to the presenity table to the publ_cache.  
```

Comment:
"presenity" miss-spelled. "publ_cache" - isn't that a code name? Can you express it in normal language?